core/src/com/biglybt/core/history/impl/DownloadHistoryManagerImpl.java

```diff
@@ -770,29 +769,17 @@ public void perform(TimerEvent event) {
 
 		// System.out.println( "loading " + file );
 
-		try{
-			if ( FileUtil.resilientConfigFileExists( file )){
-
-				Map map = FileUtil.readResilientConfigFile( file );
```

Comment:
I believe this is the instance you are referring to. 
These two FileUtil methods seems to have a failsafe implementation. The javadoc states so as well:
```
	/**
	 *
 	 * @return Map read from config file, or empty HashMap if error
	 */
```